client/src/views/SingleJob.vue

```diff
@@ -1,20 +1,16 @@
 <template>
   <div>
     <Header />
-    <SingleJob/>
   </div>
 </template>
 
 <script>
 import Header from "../components/Header.vue";
-import SingleJob from "../components/SingleJob.vue";
-import Footer from "../components/Footer.vue";
 
 export default {
   name: "SingleJob",
```

Comment:
I didn't understand what do you mean by singleJob component..  what component are we going to add here?